script/Create3Factory.s.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
```

Comment:
Should delete the entire script directory, since there is nothing relevant in there right now.